SAPHana/man/SAPHanaSR.py.7

```diff
@@ -58,22 +64,26 @@ Usage, syntax or execution errors.
 .PP
 * Example for specific permission in /etc/sudoers.d/SAPHanaSR .
 .br
-HANA 1.0 Scale-Out, SID is SLE. The attribute contains the lowercase-SID.
+HANA 1.0 scale-out, SID is SLE. The attribute contains the lowercase-SID.
 Access is allowed to the specified SID's srHook attribute only.
-Access should be always limited to the specific SID. 
+Access should be always limited to the specific SID.
+This example is similar to those given in documentation for former versions of the scale-out srHook script. In-place upgrade of the package without adding multi-target support is possible, even if the existing sudoer permission does not allow new auxiliary attributes.
 .PP
 .RS 2
 # SAPHanaSR-ScaleOut old-style needs for the srHook script.
 .br
 sleadm ALL=(ALL) NOPASSWD: /usr/sbin/crm_attribute -n hana_sle_glob_srHook -v *
 .RE
 .PP
+Note: In former documentation the central /etc/sudoers has been used instead of /etc/sudoers.d/SAPHanaSR. See sudoers(5) for details on including files. 
+.PP
 * Example for generic permission in /etc/sudoers.d/SAPHanaSR .
 .br
-HANA 1.0 Scale-Out, SID is SLE. The attribute contains the lowercase-SID. 
+HANA 1.0 and HANA 2.0 scale-out, SID is SLE. The attribute contains the lowercase-SID. 
```

Comment:
Still HANA 1.0? I would like to drop that as it is not any longer in a supported status.